internal/pkg/utils/utils_test.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 var testAnalyzer = &analysis.Analyzer{
-	Name:       "domination",
+	Name:       "dereference",
```

Comment:
This rename is outside the scope of the commit message.  Please either update the commit message or separate this to another PR.

Reply:
Reverted.